src/main/java/com/alexbro/onlinebank/facade/account/DefaultAccountFacade.java

```diff
@@ -31,6 +34,7 @@ public void transfer(String accountCode, Long cardNumber, BigDecimal sum) {
                         ACCOUNT_IS_NOT_FOUND_MESSAGE)));
 
         validateAccountFromMoney(accountFrom.getMoney(), sum);
+        validateSumLimit(sum);
```

Comment:
sum validation should be before the account fetching! If sum is invalid flow should be interrupted!